CLEO4/CCustomOpcodeSystem.cpp

```diff
@@ -1698,7 +1700,7 @@ namespace CLEO
 	{
 		DWORD hVehicle;
 		*thread >> hVehicle;
-		auto id = reinterpret_cast<CVehicleModelInfo*>(Models[GetVehiclePool().GetAt(hVehicle)->GetModelIndex() - 400])->vehicleHandlingID;
+		auto id = reinterpret_cast<CVehicleModelInfo*>(Models[GetVehiclePool().GetAt(hVehicle)->GetModelIndex()])->vehicleHandlingID;
```

Comment:
Just taking a note: Should update the develop branch accordingly later (due to the plugin-sdk patch)